### PR TITLE
VS-1024 beta workflow for exomes

### PR DESCRIPTION
**Pull request by gbggrant**
This PR Creates and exposes the `is_wgs` parameter in the GvsJointVariantCalling wdl.
It follows the rules defined in the ticket VS-1020 as to how to set `is_wgs` and the `optional interval_list` and `interval_weights_bed` inputs

### CODE REVIEW

**Comment by gbggrant** [comment]
Successful Joint Calling workflow (Exome) [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Exome%20Beta%20Test%20ggrant/job_history/0ff13881-727f-4fdf-bde7-904559eac58f).

File: scripts/variantstore/beta_docs/gvs-exome-workspace-description.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+### Genomic Variant Store Exomes workflow (beta) for variant discovery
```
Comment (gbggrant) [inline scripts/variantstore/beta_docs/gvs-exome-workspace-description.md:1]: Note - this document needs further additions / updates probably done by @kbergin

File: scripts/variantstore/beta_docs/gvs-exome-workspace-description.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+| Number of Samples | Wall Clock Time (hh:mm) | Cost $     | Cost per Sample |
+|-------------------|-------------------------|------------|-----------------|
+| 10                | 04:30                   | $1.35      | $0.14           |
+| 1000              | 07:24                   | $59.64     | $0.06           |
+| 2500              | 08:45                   | $141.28    | $0.06           |
+| 5000              | 12:00                   | $286.71    | $0.06           |
+| 10000             | 13:41                   | $604.97    | $0.06           |
+| 25000             | 63:35*                  | $1,282.65  | $0.051          |
+
+*Our test of 25,000 genomes hit some cloud turbulence and took longer than it should have. As it is an expensive test at this scale, we will update this chart when we have a need to run it again.
```
Comment (kbergin) [inline scripts/variantstore/beta_docs/gvs-exome-workspace-description.md:98]: just a note that we'll want to remove this line for exomes

File: scripts/variantstore/beta_docs/gvs-exome-workspace-description.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+---
+
+**Important Update:**
+
+Starting September 1 2023, variants in GVS are filtered using the GATK Variant Extract-Train-Score toolchain with an isolation-forest model by default. GVS maintains the ability to run VQSR on callsets up to 10,000 genomes to reproduce the same results from past analyses. See the [release notes](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/docs/release_notes/VETS_Release.pdf) for more information.
+
+---
+
+### What data does it require as input?
```
Comment (kbergin) [inline scripts/variantstore/beta_docs/gvs-exome-workspace-description.md:29]: I think this section should mention that it uses xyz interval list for exomes by default, and the linked documentation 'run your own samples' can give a bit more detail about where a readme is about that interval list. I'll update the linked documentation when I do the docs update for exomes.

**Review by kbergin — CHANGES_REQUESTED** [review]

**Review by mcovarr — APPROVED** [review]

**Review by kbergin — APPROVED** [review]
Thanks for updating that!

**Pull request merged**